en/flight_controller/pixhack_v5.md

```diff
@@ -0,0 +1,109 @@
+# Pixhack v5
+
+*Pixhack v5*<sup>&reg;</sup> is an advanced autopilot designed and made in CUAV<sup>&reg;</sup> . 
+The board is  is based on the [Pixhawk-project](https://pixhawk.org/) **FMUv5** open hardware design. It runs PX4 on the [NuttX](http://nuttx.org) OS, and is fully compatible with both PX4 <sup>&reg;</sup>  firmware. intended primarily for academic and commercial develop.
```

Comment:
There are some typographic errors here. Can you please replace this section with:
```
The board is based on the [Pixhawk-project](https://pixhawk.org/) **FMUv5** open hardware design. 
It runs PX4 on the [NuttX](http://nuttx.org) OS, and is fully compatible with PX4 firmware. 
It is intended primarily for academic and commercial developers.
```